core/client/fs/src/main/java/alluxio/client/block/stream/BlockWorkerDataWriter.java

```diff
@@ -62,14 +62,14 @@ public static BlockWorkerDataWriter create(final FileSystemContext context, long
     AlluxioConfiguration conf = context.getClusterConf();
     int chunkSize = (int) conf.getBytes(PropertyKey.USER_LOCAL_WRITER_CHUNK_SIZE_BYTES);
     long reservedBytes = Math.min(blockSize, conf.getBytes(PropertyKey.USER_FILE_RESERVED_BYTES));
-    BlockWorker blockWorker = context.getProcessLocalWorker()
```

Comment:
Might worth taking a look at whether you can change FileSystemContext to keep `BlockStore` instead of `BlockWorker`.

Reply:
Cood call, I will make it in another PR, thanks!